tests/acceptance/features/apiAuthApp/resources.feature

```diff
@@ -0,0 +1,57 @@
+Feature: create auth-app token
+  As a user
+  I want to create auth-app Tokens
+  So that I can let 3rd party apps use my account
+
+  Background:
+    Given user "Alice" has been created with default attributes
+
+
+  Scenario: admin creates user using auth-app token
+    And user "Admin" has created auth-app token with expiration time "1h" using the auth-app API
+    When the administrator creates user "Brian" using last created auth-app token
```

Comment:
can we do similar to `using "LOCAL" server`. The implementation should then return the auth-token when the password is requested for a user instead of actual password.
```suggestion
    And user "Admin" has created auth-app token with expiration time "1h" using the auth-app API
    And using auth-app token
    When the administrator creates user "Brian" using last created auth-app token
```